utils/path.js

```diff
@@ -3,6 +3,7 @@ const path = {
   auth: '/api/auth',
   hotel: '/api/hotel',
   subscribe: '/api/subscription',
+  getStaff: 'https://web-chat-labs.herokuapp.com/api/users?hotel_id=',
 };
```

Comment:
A suggestion - move the URL `https://web-chat-labs.herokuapp.com` to a separate `url.js` file inside utils folder and use that and path for `/api/users` with query string from `path.js`. The reasoning is to keep separated concerns in separated files (url, path).